modules/development/packages.nix

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache 2.0
+{pkgs, ...}: {
+  environment.systemPackages = with pkgs; [
+    # For lspci:
```

Comment:
Great, I was thinking this just yesterday and though of having git and emacs from `development/packages.nix` defined for on-host development. Now we will have place for defining those.